### PR TITLE
[new release] fileutils (0.6.5)

### PR DESCRIPTION
**Pull request by gildor478**
API to manipulate files (POSIX like) and filenames

- Project page: <a href="https://github.com/gildor478/ocaml-fileutils">https://github.com/gildor478/ocaml-fileutils</a>
- Documentation: <a href="https://gildor478.github.io/ocaml-fileutils/">https://gildor478.github.io/ocaml-fileutils/</a>

### Fixed

- `cmp` returns None when comparing two identical files and add some tests for
  that. (thanks to jamesjer).
- require OCaml > 4.14, which is now the default in Debian stable.

### CODE REVIEW

**Comment by gildor478** [comment]
Summary of failures from opam-ci:
- tezos-baking-alpha.16.0: `The test is taking more than XX seconds`
- tezos-baking-017-PtNairob.17.3: `unexpected injection on the node A, level = 1 / round = 2`
- ppx_rapper_lwt.3.1.0, ppx_rapper_lwt.3.0.0, ppx_rapper_async.3.1.0, ppx_rapper_async.3.0.0: some diff test failing in `File "test/ppx/test.expected.ml", line 1`
- tar-mirage.2.0.0: `Failed to get sources of io-page.2.3.0`
- orrandomForest.3.0.0: `[ERROR] The installation of orrandomForest failed at "R CMD BATCH install_randomForest.r".`
- oplsr.8.0.1: `[ERROR] The installation of oplsr failed at "R CMD BATCH install_pls.r".`
- octez-protocol-alpha-libs.19.0: `the test is taking longer than 60 seconds`
- octez-protocol-018-Proxford-libs.19.1, octez-protocol-018-Proxford-libs.18.0: `unexpected injection on the node A, level = 1 / round = 2`
- octez-protocol-017-PtNairob-libs.18.1: `unexpected injection on the node A, level = 4 / round = 0`
- mlmpfr.* `The compilation of mlmpfr.4.0.2 failed at "./test_installed".`
- links-mysql.0.9.8: `mysql8_stubs.c:221:3: error: unknown type name 'bool'` and `mysql8_stubs.c:192:32: error: 'MYSQL_OPT_SSL_MODE' undeclared (first use in this function); did you mean 'MYSQL_OPT_SSL_CRL'?`
- lilv.0.1.0: `Fatal error: exception Dl.DL_error("./inspect.exe: undefined symbol: lilv_node_free")`
- libssh.0.1: `Stream.of_channel` in oasis's setup.ml
- libirmin.*: `Cancelling: Timeout (120.0 minutes)`
- hacl-star.0.1.1, hacl-star.0.1: `Fatal error: exception Failure("[Hacl.ECDSA Test 1] Failure: Verification")`
- dream.*: `Error: Library "dream-httpaf.dream-websocketaf" not found.`
- docteur.0.0.1: `test_zeropage.c:39:5: error: array subscript 0 is outside array bounds of 'uint64_t[0]' {aka 'long unsigned int[]'} [-Werror=array-bounds] (score = 30)`
- cca.*: `Installation of cca.0.7 failed / Some files in /home/opam/.opam/4.14/.opam-switch/install/cca.install couldn't be installed:`
- bap-ghidra.*: `lib/bap_ghidra/ghidra_disasm.cpp:10:10: fatal error: ghidra/loadimage.hh: No such file or directory`
- FPauth-strategies.1.0.0: `exception Dl.DL_error("./unit.exe: undefined symbol: argon2_type2string")`
- FPauth-responses.1.0.0: `File "src/http/dune", line 21, characters 2-32:  dream-httpaf.dream-websocketaf`
- FPauth-core.1.0.0: `File "src/http/dune", line 21, characters 2-32: dream-httpaf.dream-websocketaf`


Overall, I don't think they have anything to do with fileutils directly.

File: packages/fileutils/fileutils.0.6.5/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "ocaml" {>= "4.14"}
+  "base-unix"
+  "stdlib-shims"
```
Comment (yawaramin) [inline packages/fileutils/fileutils.0.6.5/opam:18]: Are stdlib-shims and seq needed for OCaml 4.14? I think they are needed only for very old OCaml versions?

Comment (gildor478) [reply packages/fileutils/fileutils.0.6.5/opam]: I have removed them.

**Comment by gildor478** [comment]
Dropped in favor of v0.6.6

**Pull request closed without merging**